include/fmt/chrono.h

```diff
@@ -871,13 +894,13 @@ struct chrono_formatter {
   void write_pinf() { std::copy_n("inf", 3, out); }
   void write_ninf() { std::copy_n("-inf", 4, out); }
 
-  void format_localized(const tm& time, const char* format) {
+  void format_localized(const tm& time, const char_type* format) {
```

Comment:
Is this actually needed? Locale is not parameterized by char type.

Reply:
Good point, I've reverted it.

Reply:
After looking at this a bit more and understanding it better I think using the `put(iter_type, std::ios_base&, char_type fill, const std::tm* t, char, char)` overload is preferable here, as the overload that takes iterators for the format string needs to match the output as I did previously.